docs/en/query_language/functions/type_conversion_functions.md

```diff
@@ -17,6 +17,85 @@
 
 Converts `value` to [Decimal](../../data_types/decimal.md) of precision `S`. The `value` can be a number or a string. The `S` (scale) parameter specifies the number of decimal places.
 
+## toDecimal*OrNull
```

Comment:
aren't we just listing all functions in such cases in other articles?

Reply:
Yes, but it looks terrible for me. I have a plan to convert than long headers into short like this.

Reply:
@BayoNet then we should first check how this affects search by their name + I'm not sure if this asterisk is clear enough, maybe something like `toDecimal(32|64|128)OrNull` would be easier to spot

Reply:
It looks that search works for toDecimal(32|64|128)OrNull
![image](https://user-images.githubusercontent.com/2968367/59029150-304e7280-8866-11e9-892f-032e035d9a30.png)

Reply:
@BayoNet but it looks like this approach will force us to drop specifying arguments right in title as well (which is probably also a good thing to do), or alternatively change brackets

Reply:
I like this variant https://www.postgresql.org/docs/12/functions-sequence.html, when at the beginning of a file we have a table with functions and their parameters, but the sections with detailed description named just by functions name.

In Verica's documentation also no function parameters in headings https://www.vertica.com/docs/9.1.x/HTML/index.htm#Authoring/SQLReferenceManual/Functions/Aggregate/AggregateFunctions.htm%3FTocPath%3DSQL%2520Reference%2520Manual%7CSQL%2520Functions%7CAggregate%2520Functions%7C_____0